mcp/infra/README.md

```diff
@@ -0,0 +1,43 @@
+# Infrastructure for the MCP server
```

Comment:
how do we feel about deploying this? how likely is it that someone with a public URL to our MCP server would be able to prompt inject some write access into our concept store?

I'd recommend sticking a _big scary warning_ on this, especially as it's written (in true harrison style) so that anyone in the org should be able to deploy it!